fix(configs/kubeadm): fix SM port name

ralgozino opened the pull request:
Fix port name for kube-controller-manager and kube-scheduler in ServiceMonitor defintion.
Port name was different in the Service defintition.

Fixes #109

lnovara reviewed (changes requested):
Instead of changeing the port in the ServiceMonitor definition, I would change it in the Service since those are maintained by us while SMs are from upstream.

ralgozino commented:
> Instead of changeing the port in the ServiceMonitor definition, I would change it in the Service since those are maintained by us while SMs are from upstream.

makes sense, I've reverted the SM and changed the Service instead.
